cmd/endpoint.go

```diff
@@ -132,7 +132,11 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 			return ep, err
 		}
 	} else {
-		u = &url.URL{Path: path.Clean(arg)}
```

Comment:
why is this change gone? if need this to indicate that we treating the `arg` as Path only. `u` will be `nil`. 

```
return Endpoint{
                URL:     u,
                IsLocal: isLocal,
        }, nil
```